### PR TITLE
April 2020 news

### PR DESCRIPTION
**Pull request by aeslaughter**
@idaholab/moose-ccb Please add your contributions for April 2020
(refs #11446)

If anyone is interested I added a `git news` command.

**gitconfig**
```
[alias]
   news = !git-news.py
```

**git-news.py**
```
#!/usr/bin/env python
import sys
import subprocess
import datetime

today = datetime.date.today()
month, year = (today.month - 1, today.year) if today.month > 1 else (12, today.year-1)
after = datetime.date(year, month, 1).strftime('%Y-%m-%d')
before = datetime.date(today.year, today.month, 1).strftime('%Y-%m-%d')

author = subprocess.check_output(['git', 'config', 'user.name'], encoding='utf8').strip('\n')
cmd = ['git', 'log', '--author', author, '--after', after, '--before', before]
print(' '.join(cmd))
subprocess.call(cmd)
```

### CODE REVIEW

**Comment by aeslaughter** [comment]
@zachmprince Please add your improvements to PolynomialChaos from March

**Comment by permcody** [comment]
Nice news command! You might want to add `--no-merges` to clean it up just a bit more.

**Comment by aeslaughter** [comment]
> Nice news command! You might want to add `--no-merges` to clean it up just a bit more.

Should I just put this script in MOOSE?

File: modules/doc/content/newsletter/2020_04.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+
+## SoftwareX Paper Published
+
+The new MOOSE paper was published in SoftwareX: [MOOSE: Enabling massively parallel multiphysics simulation](https://doi.org/10.1016/j.softx.2020.100430).
+
+## Update `validParams`
+
+The framework and all modules have been updated to use a static function for `validParams`. Please
+update your applications to use the new static function and see
+[#14135](https://github.com/idaholab/moose/issues/14135) see the changes in MOOSE.
```
Comment (bwspenc) [inline modules/doc/content/newsletter/2020_04.md:11]: The issue referenced here doesn't actually give any guidance about what an app developer is supposed to do. I assume we are supposed to only declare and implement the static `validParams()` method, get rid of the templated version, and also get rid of the forward declaration of the current class (which is generally only needed for the templated `validParams()`). We should just provide those details right here.

Comment (aeslaughter) [reply modules/doc/content/newsletter/2020_04.md]: I was just lazy, I added a detailed explanation.

**Comment by aeslaughter** [comment]
@idaholab/moose-team Please add your contributions for March; I would like to get this out by the end of the week.

**Comment by lindsayad** [comment]
I don't have anything significant

File: modules/doc/content/newsletter/2020_04.md (hunk 1 above)
Comment (bwspenc) [reply modules/doc/content/newsletter/2020_04.md]: Thanks, very helpful now.

**Comment by fdkong** [comment]
May I have one minute? Let me check if I will add something

**Comment by fdkong** [comment]
OK, I am done

**Review by permcody — APPROVED** [review]

**Pull request merged**